test/unit/coffee/ConnectedUsersManagerTests.coffee

```diff
@@ -124,7 +124,7 @@ describe "ConnectedUsersManager", ->
 
 		it "should return a connected user if there is a user object", (done)->
 			cursorData = JSON.stringify(cursorData:{row:1})
-			@rClient.hgetall.callsArgWith(1, null, {connected_at:new Date(), cursorData})
+			@rClient.hgetall.callsArgWith(1, null, {connected_at:new Date(), user_id: @user._id, last_updated_at: "#{Date.now()}", cursorData})
```

Comment:
Perhaps add a new test case, `"should return a not connected user if there is a user object but has no 'user_id'"` ?

Reply:
Yes, I have a separate PR to clean this up more but I won't deploy that today.

Reply:
great 👍